mantidproject/mantid
# Display Fit result workspaces in the fit property browser

**Description of work.**
This PR adds the functionality to view the result workspaces from the fit property browser after a fit has been completed. This is to make it easier for the users so they don't have to switching between a fit window and the main workbench.

**To test:**
Plot a workspace and open the fit browser
Add a function and do a fit - a list should appear in the fit browser containing the 3 result workspaces.
Click on each of these and the workspace should be displayed.

Fixes #24477 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
